server/Application/Application.CQRS/Queries/GetExampleEntitiesPagedQuery.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Ardalis.GuardClauses;
+using Domain.Entities;
+using FluentValidation;
+
+namespace Application.CQRS.Queries;
+
+public sealed class GetExampleEntitiesPagedQuery : IQuery<OneOf<IPagedData<ExampleEntityDto>, ErrorDetails>>
+{
+    public GetExampleEntitiesPagedQuery(string? name, string? continuationToken = null)
+    {
+
+        Name = name;
+        ContinuationToken = continuationToken;
+    }
+
+    public string? ContinuationToken { get; }
+    public string? Name { get; }
+}
+
+public sealed class GetExampleEntitiesPagedQueryValidator : AbstractValidator<GetExampleEntitiesPagedQuery>
+{
+    public GetExampleEntitiesPagedQueryValidator()
```

Comment:
This validator doesn't appear to be used anywhere. If it's not needed, remove it.